api/types/installers/agentless-installer.sh.tmpl

```diff
@@ -12,17 +12,23 @@ set -o nounset
 
   . /etc/os-release
 
+  REPO_CHANNEL="{{ .RepoChannel }}"
+  if [ -n "{{ .MajorVersion }}" ]
+  then
+      REPO_CHANNEL="stable/{{ .MajorVersion }}"
+  fi
```

Comment:
I don't think this change is actually needed? We only want to populate the MajorVersion template variable so users can use it in their own scripts, but we don't need to actually use it in our scripts?